tests/integration/test_version.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of starcraft.
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License version 3, as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Starcraft versioning tests."""
```

Comment:
oh I originally thought that the starbase tests would stay there (and we would remove them when merging), that's why I suggested putting them in a separate subdir. I assume you moved this manually, so do you think this test is worth keeping?

Reply:
I agree with starbase tests like `test_init` not needing to be copied over, but the version tests seem relevant.

What about reorganizing the starbase tests to:
```
tests/
├── __init__.py
├── integration
│   ├── __init__.py
│   ├── starbase
│   │   ├── __init__.py
│   │   └── test_init.py
│   └── test_version.py
└── unit
    ├── __init__.py
    └── starbase
        ├── __init__.py
        └── test_init.py
```

Reply:
I agree with with @mr-cal — this particular integration test seems like a nice thing to have even in the downstream uses.

On the flip side, given that we now have something in `tests/integration/` I'm not sure whether we need to keep the other test file at all.